### PR TITLE
qa/tests: added 14.2.2 to the mix

### PR DESCRIPTION
**Pull request by yuriw**
Fixes: http://tracker.ceph.com/issues/40251
Fixes: http://tracker.ceph.com/issues/40832
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com:80/yuriw-2019-07-19_15:52:38-upgrade:nautilus-p2p-nautilus-distro-basic-smithi/

File: qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ overrides:
     - filesystem is degraded
     - OBJECT_MISPLACED
     ### ref: https://tracker.ceph.com/issues/40251 
-    - failed to encode map 
+    #removed seee ^ - failed to encode map
```
Comment (neha-ojha) [inline qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml]: s/seee/see

File: qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml [hunk 2]
```diff
@@ -84,17 +84,19 @@ tasks:
 - print: "**** done workload v14.2.0"
 
 
-# v14.2.1
+# v14.2.1 emoved per http://tracker.ceph.com/issues/40251
```
Comment (neha-ojha) [inline qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml]: s/emoved/removed

File: qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml [hunk 3]
```diff
@@ -128,7 +130,8 @@ workload:
 workload_nautilus:
    full_sequential:
    - workunit:
-       tag: v14.2.0
+       branch: nautilus
+       #tag: v14.2.0
```
Comment (neha-ojha) [inline qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml]: did you mean to remove this?

Comment (yuriw) [reply qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml]: Yes, it feels wrong, agree?

**Comment by neha-ojha** [comment]
> http://pulpito.front.sepia.ceph.com:80/yuriw-2019-07-19_15:52:38-upgrade:nautilus-p2p-nautilus-distro-basic-smithi/

The reason we are seeing the "failed to encode map e99 with expected crc" failures in nautilus-p2p-parallel is because it is still installing 14.2.0, which still has the same problem. We should either continue to whitelist or just start with 14.2.2.

**Comment by yuriw** [comment]
@jdurgin @neha-ojha my preference would be to add 14.2.1 back and continue to whitelist `failed to encode map e99 with expected crc` so we test more point releases.  What say you ?

**Comment by jdurgin** [comment]
I'd rather start with 14.2.2, and not whitelist 'failed to encode map' so that we can detect if we introduce another cause for that error in a later release.

**Comment by yuriw** [comment]
removed 14.2.0 and testing 
http://pulpito.front.sepia.ceph.com:80/yuriw-2019-07-22_16:09:30-upgrade:nautilus-p2p:nautilus-p2p-parallel-nautilus-distro-basic-smithi/

**Comment by yuriw** [comment]
removed 14.2.0 and testing 
http://pulpito.front.sepia.ceph.com:80/yuriw-2019-07-22_16:09:30-upgrade:nautilus-p2p:nautilus-p2p-parallel-nautilus-distro-basic-smithi/

**Pull request merged**